samples/graph-activity-feed/csharp/README.md

```diff
@@ -0,0 +1,67 @@
+﻿# Notify Feed App
+
+Bot Framework v4 Activity Feed sample.
+
+This sample has been created using [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview?view=graph-rest-beta), it shows how trigger a Activity feed notification from your Tab, it triggers the feed notification for User, Chat and Team scope and send back to conversation.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- Run the bot from a terminal or from Visual Studio:
+
+  A) From a terminal, navigate to `samples/graph-activity-feed/csharp` folder
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/graph-activity-feed/csharp` folder
+  - Select `TabActivityFeed.csproj` file
+  - Press `F5` to run the project
+
+## Testing the bot using Bot Framework Emulator
```

Comment:
Please correct the readme file. This still talks about Bot testing. Please include details on setting up application permission used in this sample.